src/main/java/com/conveyal/gtfs/graphql/fetchers/NestedJDBCFetcher.java

```diff
@@ -11,20 +9,26 @@
 import org.slf4j.LoggerFactory;
 
 import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
 
 import static com.conveyal.gtfs.graphql.GraphQLUtil.multiStringArg;
+import static com.conveyal.gtfs.graphql.GraphQLUtil.namespacedTableFieldName;
+import static com.conveyal.gtfs.graphql.GraphQLUtil.namespacedTableName;
 import static com.conveyal.gtfs.graphql.GraphQLUtil.stringArg;
+import static com.conveyal.gtfs.graphql.fetchers.JDBCFetcher.makeInClause;
 import static graphql.schema.GraphQLFieldDefinition.newFieldDefinition;
 
 /**
- * This fetcher uses nested calls to the general JDBCFetcher for SQL data. It uses the parent entity and a series of
- * joins to leap frog from one entity to another more distantly-related entity. For example, if we want to know the
- * routes that serve a specific stop, starting with a top-level stop type, we can nest joins from stop ABC -> pattern
- * stops -> patterns -> routes (see below example implementation for more details).
+ * This fetcher creates one big query by joining tables in the order specified in the initial definition.  The data
+ * returned will only consist of the columns from the final table definition.  All other tables are assumed to be
+ * intermediate tables used to create joins to obtain filtered data from the final table.  Also, any grqphQL arguments
+ * are applied only to the final table.
  */
-public class NestedJDBCFetcher implements DataFetcher<List<Map<String, Object>>> {
+public class NestedJDBCFetcher implements DataFetcher<Object> {
```

Comment:
Again I'm hesitant to lose the type information. Using `Object` defeats the purpose of generic types. Are there cases where we're not returning lists of maps?